pom.xml

```diff
@@ -58,6 +58,11 @@
                 <artifactId>jsr305</artifactId>
                 <version>${version.jsr305.plugin}</version>
             </dependency>
+            <dependency>
+                <groupId>io.jsonwebtoken</groupId>
```

Comment:
Such dependency is already declared in [che-dependencies](https://github.com/eclipse/che-dependencies/blob/che6/pom.xml#L469-L472). Are you sure that you need the same one here?